azure_li_services/units/system_setup.py

```diff
@@ -258,6 +259,17 @@ def set_stonith_service(config):
         )
 
 
+def load_stonith_needed_modules():
+    modules = Defaults.get_stonith_needed_modules()
+    for module in modules:
+        load_module_path = '/etc/modules-load.d/{0}.conf'.format(module)
```

Comment:
I think we should have a fixed file name "azure-extra-modules" and then write all the modules into that file rather than creating a file for each module